kernel-commits.mk

```diff
@@ -1,7 +1,7 @@
-KERNEL_COMMIT_amd64_v5.10.186_generic = be89a9e63306
-KERNEL_COMMIT_amd64_v6.1.38_generic = d586ca2d3f2e
-KERNEL_COMMIT_amd64_v6.1.38_rt = 49dba60a59d9
-KERNEL_COMMIT_arm64_v5.10.104_nvidia = c21d5a4dd498
-KERNEL_COMMIT_arm64_v5.10.186_generic = 6a861e0433a6
-KERNEL_COMMIT_arm64_v6.1.38_generic = 92466d23a9bf
```

Comment:
@andrewd-zededa , please, move the kernel update to a dedicated commit, so it helps with traceability (our kernel updates are all in dedicated commits) and separates logically the changes of your PR....

Reply:
Split the kernel change out to its own commit.  I can squash the other two commits down to one if needed.